examples/adder/Cargo.toml

```diff
@@ -25,3 +26,20 @@ opt-level = "z"
 lto = true
 debug = false
 panic = "abort"
+
+[package.metadata.near.build]
+# This is the Docker metadata line:
+image = "nearprotocol/contract-builder:latest-amd64"
+
+[package.metadata.near.reproducible_build]
+# docker image, descriptor of build environment
+image = "nearprotocol/contract-builder:latest-amd64"
+# tag after colon above serves only descriptive purpose; image is identified by digest
+image_digest = "sha256:53b34ea28b8995c6e8908e76db8ea0b117ac73bc91c83c8aae930e635e9ef885"
+container_build_command = [
+    "cargo",
+    "near",
+    "build",
+    "reproducible-wasm",
+    "--no-lockedd",
+]
```

Comment:
i'd replace the command (inside docker container) everywhere in Cargo.toml-s with 
what is specified in default template https://github.com/near/cargo-near/blob/main/cargo-near/src/commands/new/new-project-template/Cargo.template.toml#L29-L35. 

`cargo near build reproducible-wasm --no-locked` is meant to be run on host (outside docker container). It just runs a container from docker `image` specified and then `container_build_command` command inside that container. 
Running `reproducible-wasm` as  `container_build_command` will just attempt to run docker from inside docker container, and will just usually be an error as the result.